### PR TITLE
[zh]Update content/zh/examples/application/mysql/mysql-statefulset.yaml

### PR DESCRIPTION
**Pull request by kinzhi**

### CODE REVIEW

File: content/zh/examples/application/mysql/mysql-statefulset.yaml [hunk 1]
```diff
@@ -47,7 +47,7 @@ spec:
           set -ex
           # Skip the clone if data already exists.
```
Comment (tengqm) [inline content/zh/examples/application/mysql/mysql-statefulset.yaml:48]: 如果时间允许，请翻译这些注释

Comment (kinzhi) [reply content/zh/examples/application/mysql/mysql-statefulset.yaml]: 已经翻译，请teng老师帮忙再看看

File: content/zh/examples/application/mysql/mysql-statefulset.yaml [hunk 2]
```diff
@@ -21,13 +21,14 @@ spec:
         - "-c"
         - |
           set -ex
-          # Generate mysql server-id from pod ordinal index.
+          # 以 Pod 序号索引生成 Mysql 服务器ID。
```
Comment (tengqm) [inline content/zh/examples/application/mysql/mysql-statefulset.yaml:24]:
```suggestion
          # 基于 Pod 序号生成 MySQL 服务器的 ID。
```

File: content/zh/examples/application/mysql/mysql-statefulset.yaml [hunk 3]
```diff
@@ -21,13 +21,14 @@ spec:
...
           [[ `hostname` =~ -([0-9]+)$ ]] || exit 1
           ordinal=${BASH_REMATCH[1]}
           echo [mysqld] > /mnt/conf.d/server-id.cnf
-          # Add an offset to avoid reserved server-id=0 value.
+          # 添加偏移量以避免使用 server-id=0 的值。
```
Comment (tengqm) [inline content/zh/examples/application/mysql/mysql-statefulset.yaml:28]:
```suggestion
          # 添加偏移量以避免使用 server-id=0 这一保留值。
```

File: content/zh/examples/application/mysql/mysql-statefulset.yaml [hunk 4]
```diff
@@ -21,13 +21,14 @@ spec:
...
           echo server-id=$((100 + $ordinal)) >> /mnt/conf.d/server-id.cnf
           # Copy appropriate conf.d files from config-map to emptyDir.
+          # 将合适的 conf.d 文件从 config-map 复制到 emptyDir 。
```
Comment (tengqm) [inline content/zh/examples/application/mysql/mysql-statefulset.yaml:31]:
```suggestion
          # 将合适的 conf.d 文件从 config-map 复制到 emptyDir。
```

File: content/zh/examples/application/mysql/mysql-statefulset.yaml [hunk 5]
```diff
@@ -45,15 +46,15 @@ spec:
         - "-c"
         - |
           set -ex
-          # Skip the clone if data already exists.
+          # 如果已有数据，则跳过克隆。
           [[ -d /var/lib/mysql/mysql ]] && exit 0
-          # Skip the clone on primary (ordinal index 0).
+          # 跳过主实例(序号索引0)的克隆。
```
Comment (tengqm) [inline content/zh/examples/application/mysql/mysql-statefulset.yaml:51]:
```suggestion
          # 跳过主实例（序号索引 0）的克隆。
```

File: content/zh/examples/application/mysql/mysql-statefulset.yaml [hunk 6]
```diff
@@ -45,15 +46,15 @@ spec:
...
           [[ `hostname` =~ -([0-9]+)$ ]] || exit 1
           ordinal=${BASH_REMATCH[1]}
           [[ $ordinal -eq 0 ]] && exit 0
-          # Clone data from previous peer.
+          # 从备份节点克隆数据。
```
Comment (tengqm) [inline content/zh/examples/application/mysql/mysql-statefulset.yaml:55]:
```suggestion
          # 从原来的对等节点克隆数据。
```

File: content/zh/examples/application/mysql/mysql-statefulset.yaml [hunk 7]
```diff
@@ -88,7 +89,7 @@ spec:
           timeoutSeconds: 5
         readinessProbe:
           exec:
-            # Check we can execute queries over TCP (skip-networking is off).
+            # 检查我们是否可以通过TCP执行查询(跳过网络是关闭的)。
```
Comment (tengqm) [inline content/zh/examples/application/mysql/mysql-statefulset.yaml:92]:
```suggestion
            # 检查我们是否可以通过 TCP 执行查询（skip-networking 是关闭的）。
```

File: content/zh/examples/application/mysql/mysql-statefulset.yaml [hunk 8]
```diff
@@ -105,22 +106,22 @@ spec:
           set -ex
           cd /var/lib/mysql
 
-          # Determine binlog position of cloned data, if any.
+          # 确定克隆数据的binlog位置(如果有的话)。
```
Comment (tengqm) [inline content/zh/examples/application/mysql/mysql-statefulset.yaml:109]:
```suggestion
          # 确定克隆数据的 binlog 位置（如果有的话）。
```

File: content/zh/examples/application/mysql/mysql-statefulset.yaml [hunk 9]
```diff
@@ -105,22 +106,22 @@ spec:
...
           if [[ -f xtrabackup_slave_info && "x$(<xtrabackup_slave_info)" != "x" ]]; then
-            # XtraBackup already generated a partial "CHANGE MASTER TO" query
-            # because we're cloning from an existing replica. (Need to remove the tailing semicolon!)
+            # XtraBackup 已经生成了部分的 “CHANGE MASTER TO” 查询
+            # 因为我们从一个现有副本进行克隆。(需要删除尾号!)
```
Comment (tengqm) [inline content/zh/examples/application/mysql/mysql-statefulset.yaml:112]:
```suggestion
            # 因为我们从一个现有副本进行克隆。(需要删除末尾的分号!)
```

File: content/zh/examples/application/mysql/mysql-statefulset.yaml [hunk 10]
```diff
@@ -105,22 +106,22 @@ spec:
...
             cat xtrabackup_slave_info | sed -E 's/;$//g' > change_master_to.sql.in
-            # Ignore xtrabackup_binlog_info in this case (it's useless).
+            #忽略 xtrabackup_binlog_info 在这种情况下(它是没用的)。
```
Comment (tengqm) [inline content/zh/examples/application/mysql/mysql-statefulset.yaml:114]:
```suggestion
            # 在这里要忽略 xtrabackup_binlog_info （它是没用的）。
```

File: content/zh/examples/application/mysql/mysql-statefulset.yaml [hunk 11]
```diff
@@ -105,22 +106,22 @@ spec:
...
             rm -f xtrabackup_slave_info xtrabackup_binlog_info
           elif [[ -f xtrabackup_binlog_info ]]; then
-            # We're cloning directly from primary. Parse binlog position.
+            # 我们直接从主实例进行克隆。解析binlog位置。
```
Comment (tengqm) [inline content/zh/examples/application/mysql/mysql-statefulset.yaml:117]:
```suggestion
            # 我们直接从主实例进行克隆。解析 binlog 位置。
```

File: content/zh/examples/application/mysql/mysql-statefulset.yaml [hunk 12]
```diff
@@ -105,22 +106,22 @@ spec:
...
             [[ `cat xtrabackup_binlog_info` =~ ^(.*?)[[:space:]]+(.*?)$ ]] || exit 1
             rm -f xtrabackup_binlog_info xtrabackup_slave_info
             echo "CHANGE MASTER TO MASTER_LOG_FILE='${BASH_REMATCH[1]}',\
                   MASTER_LOG_POS=${BASH_REMATCH[2]}" > change_master_to.sql.in
           fi
 
-          # Check if we need to complete a clone by starting replication.
+          # 检查我们是否需要通过启动来完成克隆。
```
Comment (tengqm) [inline content/zh/examples/application/mysql/mysql-statefulset.yaml:124]:
```suggestion
          # 检查我们是否需要通过启动复制来完成克隆。
```

**Comment by tengqm** [comment]
Again, please squash your PR into one commit.

**Comment by kinzhi** [comment]
Please help to review this PR @ @tengqm

**Pull request merged**